test/unit/fh3/resources/test_resources.js

```diff
@@ -37,5 +40,12 @@ module.exports = {
       assert.equal(data.state,"RUNNING");
       return cb();
     });
+  },
+  'fhc resources list --env': function(cb) {
+    listResourcesCmd({env:'dev'}, function(err) {
+      assert.equal(err, null);
```

Comment:
Since the REST service is not working as expected we are not able to get the real result to increase the validations at this point. Are you happy in we move one with the simple unit test and improve it in the future when the service be fixed?